.github/workflows/golangci-lint.yaml

```
@@ -7,9 +7,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
```

should this be 1.18?
```suggestion
          go-version: 1.18
```

---

we probably want to run CI on the lowest version we can, to make sure we catch issues for people running older versions of go